docs/release-notes/Percona Everest 0.8.0 (2024-02-21).md

```diff
@@ -0,0 +1,21 @@
+# What's new in Percona Everest 0.8.0
+
+To begin your journey with Percona Everest, check out the [Quickstart Guide for Percona Everest](../quickstart-guide/quick-install.md).
+
+Percona Everest is an open source private database-as-a-service that helps developers deploy code faster, scale deployments rapidly, and reduce database administration overhead. Plus, you can regain control over your data, database configuration, and DBaaS costs.
+
+Version 0.8.0 introduces the following changes:
```

Comment:
will there be something about multinamespaces?

Reply:
Yes.